composer.json

```diff
@@ -21,13 +21,5 @@
         "psr-4": {
             "Extend\\Integration\\": ""
```

Comment:
What about this above line? I know nothing about composer but I would assume there shouldn't be any references to `Extend\Integration`

Reply:
That says that it an alias helloextend/integration.  All the module's class names still start with Extend\Integration, according to PSR4 standards

Reply:
See a similar example here https://github.com/magento/magento2/blob/a047c7639c63bf2b858e0bbd9689eb993a48a5a2/app/code/Magento/Catalog/composer.json#L51

Reply:
^even though in composer, it's installed in the folder magento/module-catalog